modules/ebi-metagenomics/hhsuite/buildhhdb/meta.yml

```diff
@@ -0,0 +1,46 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hhsuite_buildhhdb"
+description: create an HH database to use for hhblits searches
+keywords:
+  - HH
+  - model
+  - database
+tools:
+  - "hhsuite":
+      description: "HH-suite3 for fast remote homology detection and deep protein annotation"
+      homepage: "https://github.com/soedinglab/hh-suite"
+      documentation: "https://github.com/soedinglab/hh-suite/wiki"
+      tool_dev_url: "https://github.com/soedinglab/hh-suite"
+      doi: "10.1371/journal.pone.0082138"
+      licence: ["GPL v3"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - a3m:
+      type: directory
+      description: Directory containing a3m format MSAs
+      pattern: "*"
```

Comment:
There's another pattern for directory here I missed last time

Reply:
Which I also missed :P